### ISSUE TITLE
Version field bundle is not supported

### ISSUE THREAD

**Issue by sciamannikoo** [nepda/youtrack-client#34]
To get a version field bundle, the API call is as described [here](https://www.jetbrains.com/help/youtrack/incloud/2017.1/Get-a-Version-Bundle.html).

I'm using `\YouTrack\Connection::getBundle` and "version" as first argument, and the name of the bundle as a second argument.

I see this method is trying to find a `YouTrack\VersionBundle` class, which does not exist.

The only "bundle" classes are for builds, enum, and owned fields.

I also see that in `\YouTrack\Connection::$bundle_paths`, three bundle types are explicitly commented out:
```
/*
'state' => 'stateBundle',
'version' => 'versionBundle',
'user' => 'userBundle'
*/
```

So, how I'm supposed to get the bundles of a version field?

**Comment by nepda** [comment]
I don't use this part of the client library by my self. Thank you for your effort! I hope i can spend some time on the weekend for this feature.

**Comment by nepda** [comment]
Released v1.6.0 Wit version bundle support. Thank you! Please report any problems :)
